E2E-Demos/Investment-Banking/Airflow/spark_etl_new.yaml

```diff
@@ -2,63 +2,45 @@ apiVersion: "sparkoperator.hpe.com/v1beta2"
 kind: SparkApplication
 metadata:
   name: "spark-etl"
-  namespace: "spark"
+  namespace: "hpedemo-user01"
 spec:
   sparkConf:
-    # S3 credentials should be specified below
-    spark.hadoop.fs.s3a.endpoint: "https://objectstore-zone1-svc.dataplatform.svc.cluster.local:9000/"
-    spark.hadoop.fs.s3a.access.key: "minioadmin"
-    spark.hadoop.fs.s3a.secret.key: "minioadmin"
-    spark.hadoop.fs.s3a.path.style.access: "true"
+    spark.executorEnv.SPARK_USER: hpedemo-user01
+    spark.hadoop.fs.s3a.access.key: <access-key>
+    spark.hadoop.fs.s3a.secret.key: <secret-key>
+    spark.hadoop.fs.s3a.endpoint: "ua01.hpe-minio-ezaf.com:9000"
```

Comment:
Shouldn't it be s3 proxy endpoint?

Reply:
Yeah, agree. Here is the same comment. We need to use the secret with creds or pass s3 creds as DAG params and use here template